ajax/libs/sinon.js/package.json

```diff
@@ -16,6 +16,15 @@
     "type": "git",
     "url": "https://github.com/sinonjs/sinon"
   },
+  "npmName": "sinon",
+  "npmFileMap": [
+    {
+      "basePath": "pkg",
+      "files": [
+        "**/+(sinon.|sinon-no-sourcemaps.)*"
```

Comment:
`pkg` can be the basePath, the basePath is where the path would be treated as the root directory of a file fetching pattern by auto-updater.